nncf/quantization/layers.py

```diff
@@ -55,25 +55,29 @@ def __init__(self, num_bits: int,
                  signedness_to_force: Optional[bool],
                  narrow_range: bool,
                  scale_shape: Tuple[int, ...],
-                 logarithm_scale: bool):
+                 logarithm_scale: bool,
+                 apply_saturation_fix: bool = False):
         super().__init__(num_bits, mode, signedness_to_force, narrow_range)
         self.scale_shape = scale_shape
         self.logarithm_scale = logarithm_scale
+        self.apply_saturation_fix = apply_saturation_fix
```

Comment:
`apply_saturation_fix` - is a common parameter of QuantizerSpec for PT and TF backend. @vshampor What do you think about moving this parameter to the common part in this PR?

Reply:
I agree. I will place it to the common part

Reply:
Done